### PR TITLE
Major refactor of Communication layer

### PR DESCRIPTION
**Pull request by muzman123**
Refactored Communication layer to remove instances and classes that supported AMQP
Linked local variables for MQTT connection to config.ini.example file

### CODE REVIEW

File: src/CommunicationLayer/CommDeviceControl/CommDeviceManager.cpp [hunk 1]
```diff
@@ -16,22 +21,22 @@ void CommDeviceManager::connectToDevice(CommDefines::Type type)
...
-        InternetCommDevice* internetCommDevice = new InternetCommDevice();
-        internetCommDevice->setQueueName(queueName_);
-       // internetCommDevice->setChannel(channel_);
-        connect(internetCommDevice, &InternetCommDevice::dataReceived, this, &CommDeviceManager::handleJsonDataIncoming);
-        connect(internetCommDevice, &InternetCommDevice::finished, internetCommDevice, &QObject::deleteLater);
-        internetCommDevice->start();
+        InternetCommDevice* internetCommDevice = new InternetCommDevice(ipAddress_,port_,exchange_);
+        internetCommDevice->connectToBroker();
+        connect(internetCommDevice->getClient(), &QMqttClient::messageReceived, this, &CommDeviceManager::handleJsonDataIncoming);
+        // connect(internetCommDevice, &InternetCommDevice::finished, internetCommDevice, &QObject::deleteLater);
```
Comment (asallh) [inline src/CommunicationLayer/CommDeviceControl/CommDeviceManager.cpp:27]: Do we need this, remove if not needed.

Comment (jtran217) [inline src/CommunicationLayer/CommDeviceControl/CommDeviceManager.cpp:27]: Prob delete this

File: src/CommunicationLayer/CommunicationContainer.h [hunk 2]
```diff
@@ -9,6 +9,7 @@ class ConnectionController;
 
 class CommDeviceManager;
 class I_JsonReceiver;
+//class InternetCommDevice;
```
Comment (asallh) [inline src/CommunicationLayer/CommunicationContainer.h:12]: Remove if not needed

**Review by jtran217 — APPROVED** [review]
Overall looks solid bud

File: src/CommunicationLayer/CommunicationContainer.h [hunk 3]
```diff
@@ -17,8 +18,9 @@ class CommunicationContainer
     ~CommunicationContainer();
 
     I_JsonReceiver& jsonReceiver();
-    ConnectionController& connectionController();
+    //ConnectionController& connectionController();
     CommDeviceManager& commDeviceManager();
+    //InternetCommDevice& internetCommDevice();
```
Comment (asallh) [inline src/CommunicationLayer/CommunicationContainer.h:23]: Remove these comments if not needed, try to make the code as simple as needed

**Review by asallh — CHANGES_REQUESTED** [review]
Just remove the unnecessary comments and should be good

**Review by muzman123 — COMMENTED** [review]
removed comments

**Review by asallh — APPROVED** [review]

**Pull request merged**